EPRN.Portal/Areas/Reprocessor/Controllers/PRNSController.cs

```diff
@@ -107,13 +108,11 @@ public async Task<IActionResult> CheckYourAnswers(CheckYourAnswersViewModel chec
 
             await _prnService.SaveCheckYourAnswers(checkYourAnswersViewModel.Id);
 
-            return RedirectToAction(
-                Routes.Areas.Actions.PRNS.WhatToDo, 
-                Routes.Areas.Controllers.Reprocessor.PRNS, 
-                new 
-                { 
-                    area = Category, 
-                    id = checkYourAnswersViewModel.Id 
+            return RedirectToAction(Routes.Areas.Actions.PRNS.DestinationPrn, Routes.Areas.Controllers.Reprocessor.PRNS,
```

Comment:
When parameters go on a long list, move each one on to a new line

Reply:
Done.